staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion.go

```diff
@@ -93,3 +101,16 @@ func (c *FakePods) Evict(ctx context.Context, eviction *policy.Eviction) error {
 func (c *FakePods) ProxyGet(scheme, name, port, path string, params map[string]string) restclient.ResponseWrapper {
 	return c.Fake.InvokesProxy(core.NewProxyGetAction(podsResource, c.ns, scheme, name, port, path, params))
 }
+
+// SetContainerFakeLogs set fake logs for specified container and log content
+func (c *FakePods) SetContainerFakeLogs(container, fakeLogs string) error {
+	if container == "" {
```

Comment:
Do we only call this function in a trusted codepath? If so, I don't think we need a empty check.